docs/spec/ibc/README.md

```diff
@@ -1,3 +0,0 @@
-# Cosmos Inter-Blockchain Communication (IBC) Protocol
-
```

Comment:
I think it might be worth leaving this - we have linked to this folder before.

Reply:
all specs live in the corresponding modules now

Reply:
I realise - I mean externally (on some blog posts). Should we link to the new module specs then?